src/imgtools/cli/index.py

```
@@ -73,11 +73,15 @@ def index(
-
-    logger.info("Crawling completed.")
-    logger.info("Crawl results saved to %s", crawler.output_dir)
+    try:
+        crawler.crawl()
+    except Exception as e:
+        logger.exception("Crawling failed: %s", e)
+        # exit with a non-zero status code
+        raise click.ClickException(f"Crawling failed: {e}") from e
+    else:
+        logger.info("Crawling completed successfully.")
+        logger.info("Crawl results saved to %s", crawler.output_dir)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it

(R1720)